massiv1-2/main.cpp

```diff
@@ -0,0 +1,77 @@
+#include <iostream>
+#include <random>
+#include <ctime>
+using namespace std;
+
+
+int main() {
+  const int SIZ = 200;
+  int size, p;
+  mt19937 gen(time(nullptr));
+  cout << "enter massiv size [2," << SIZ << "]" << endl;
+  cin >> size;
+  double mas[SIZ];
+  cout << "will you enter randomly or set manually? enter 1-random, 2-manually" << endl;
+  cin >> p;
+  if (p==1) {
+	for (int i = 0; i < size; i++) {
+	  uniform_real_distribution<> urd(-100, 100);//random massiv
```

Comment:
I'd like to let user define interval boundaries

Reply:
fixed